### ISSUE TITLE
Duplicate content for /movies and /movies/1

### ISSUE THREAD

**Issue by devignerforhire** [ivandoric/Making-Websites-With-October-CMS#14]
Hi Ivan,

First off let me start by saying thank you for this awesome video series. I'm new to October and it's really helping me get my around all the essential bits.

One thing I noticed in the slugs/pagination video was that you have duplicate content for /movies and /movies/1 (which both output the same content). Do you know if October has a built in method for dealing with this, or should we setup a 301 redirect?

Keep the videos coming :)

Cheers,
Chris

Ps, I tried to post this as a comment on the specific lesson post, but was not able to comment despite logging into Disqus and Twitter.

**Comment by ivandoric** [comment]
Hi  @devignerforhire, 

yeah. There is a problem in that video. Check out the video after that to see how to fix it: https://www.youtube.com/watch?v=uCR0VV6N5FE

**Comment by devignerforhire** [comment]
@ivandoric awesome, thank you!

**Comment by devignerforhire** [comment]
I just re-watched the images/galleries video and it seems the issue you were having was that the single layout wasn't working unless you append /movie/ to the url, where as the issue I'm referring to is that the first pagination item goes to movies/1 when it really should go to /movies (hope that makes sense).

**Comment by ivandoric** [comment]
Yeah, I know what the problem was :) ... I figured it out after I recorded the video. 

But thanks anyway :)
